src/cup1d/likelihood/likelihood.py

```diff
@@ -339,19 +349,23 @@ def get_log_like(
                 print("compute chi2 for z={}".format(z))
             # get data
             p1d = self.data.get_Pk_iz(iz)
-            data_cov = self.data.get_cov_iz(iz)
+            data_icov = self.data.get_icov_iz(iz)
             # add covariance from emulator
-            cov = data_cov + self.emu_cov_factor * emu_covar[iz]
+            if self.emu_cov_factor == 0:
+                icov = data_icov
+            else:
+                icov = data_icov + np.linalg.inv(
+                    self.emu_cov_factor * emu_covar[iz]
```

Comment:
@jchavesmontero - The inverse of the sum is not the sum of the inverse!

Reply:
I would add the covariances and then invert (if emu_cov_factor > 0)

Reply:
you are right! I will fix this. The good news is that I am not using that option now